renter/proto/session.go

```diff
@@ -41,8 +45,15 @@ var (
 	// question has reached its maximum revision number, meaning the contract
 	// can no longer be revised.
 	ErrContractFinalized = errors.New("contract cannot be revised further")
+
+	lockTimeout uint64 = DefaultLockTimeout
 )
 
+// SetLockTimeout sets the timeout for lock requests to the given value.
+func SetLockTimeout(timeout uint64) {
+	lockTimeout = timeout
```

Comment:
This would cause a race condition.

Reply:
I only call this function while initializing the app, but if a user calls this from goroutines, it might be a problem.